storage/range_tree.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Bram Gruneir (bram.gruneir@gmail.com)
+
+package storage
+
+import (
+	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/storage/engine"
+	"github.com/cockroachdb/cockroach/util"
+)
+
+// SetupRangeTree creates a new range tree.  This should only be called as part of
+// store.BootstrapRange.
+func (s *Store) SetupRangeTree(batch engine.Engine, ms *engine.MVCCStats,
```

Comment:
We generally define all methods on a type in one file. I'd be willing to break with that convention for the huge Store and Range types but here it doesn't look like you're even using `s` so this can be a function instead of a method (and it doesn't look like it needs to be exported).